Get rid of css dart2 division deprecation warning

tnatanael opened the pull request:
I created this pull request to correct the usage of / division in the role code to use the new math.div function.
This will avoid any error messages of deprecation in bootstrap.
For this to work i needed to upgrade hugo to version 0.83.1 since the old version does not support @use in sass.

Commit:
Simple correction using math test

Commit:
Adding a helper div() function to use as a wrapper for math.div in sass, also updated hugo for 0.83.1 to get support for @use in sass

XhmikosR commented:
Thanks for the PR but please check your committed files. You have unrelated changes like the dist files. Also

1. Hugo is unrelated since the core CSS is built with dart-sass
2. rfs is an external project, you should probably submit these there, but I'm unsure how much traction it will get

Commit:
Removing dist files

Commit:
Merge branch 'css-dart2-division-deprecation' of github.com:tnatanael/bootstrap into css-dart2-division-deprecation

Commit:
Merge branch 'css-dart2-division-deprecation' of github.com:tnatanael/bootstrap into css-dart2-division-deprecation

Commit:
Merge branch 'css-dart2-division-deprecation' of github.com:tnatanael/bootstrap into css-dart2-division-deprecation

tnatanael commented:
> Thanks for the PR but please check your committed files. You have unrelated changes like the dist files. Also
> 
> 1. Hugo is unrelated since the core CSS is built with dart-sass
> 2. rfs is an external project, you should probably submit these there, but I'm unsure how much traction it will get

Hy! Thanks for your response, i removed the dist files from commit.

1 - Hugo is used to build the docs, but the actual binary version used is 0.71.1 witch does not suport @use, so i updated to 0.83.0 (current version), what could i do to address it?

2 - Can you explain what is the point about _rfs? Since it is in scss/vendor folder i corrected it, i could not do that? What i should do to correct it inside bootstrap?

tnatanael committed:
Merge branch 'main' into css-dart2-division-deprecation

Commit:
Merge branch 'main' of https://github.com/twbs/bootstrap into css-dart2-division-deprecation

Commit:
Lock update

Commit:
Merge branch 'css-dart2-division-deprecation' of github.com:tnatanael/bootstrap into css-dart2-division-deprecation

XhmikosR commented:
Please don't change unrelated stuff. You do not need to change the dependencies or anything.
